api/v1beta2/configuration_types.go

```diff
@@ -109,49 +109,49 @@ type Backend struct {
 	Inline string `json:"inline,omitempty"`
 
 	// Remote is needed for the Terraform `remote` backend type.
-	Remote RemoteBackendConf `json:"remote,omitempty"`
+	Remote *RemoteBackendConf `json:"remote,omitempty"`
 
 	// Artifactory is needed for the Terraform `artifactory` backend type.
-	Artifactory ArtifactoryBackendConf `json:"artifactory,omitempty"`
+	Artifactory *ArtifactoryBackendConf `json:"artifactory,omitempty"`
```

Comment:
Are these backend are belong to Remote?

Reply:
I think uses can also use these backends if they use inline HCL configuration.

Reply:
Why?

Reply:
> Are these backend are belong to Remote?

The meaning of "Remote" is the `spec.Remote`?

Reply:
You are right.